### ISSUE TITLE
Firebase images not showing

### ISSUE THREAD

**Issue by Aryqs-Ipsum** [nhn/tui.editor#1607]
## Describe the bug
Some images not showing because their url contain % characters

## To Reproduce
1. Use `addImageBlobHook` with firebase storage url `https://firebasestorage.googleapis.com/v0/b/**.appspot.com/o/images%2Fimage.png?alt=media`
2.
```json
{
  "error": {
    "code": 400,
    "message": "Invalid HTTP method/URL pair."
  }
}
```

## Expected behavior
`%2F` is replaced with a `/` and link is broken. Not all characters should be encoded.

## Screenshots
![image](https://user-images.githubusercontent.com/45895096/123516539-e75ff580-d69c-11eb-988c-b3b15fe01743.png)

## Desktop (please complete the following information):
 - OS: Windows 10
 - Browser: Chrome
 - Version: 91

**Comment by js87zz** [comment]
@Aryqs-Ipsum 
If you can't bring the image, I don't think it's an editor's issue, is there any other factor?

**Comment by Aryqs-Ipsum** [comment]
The image works. only, when I import it via the editor hook, the url is serialized.

**Comment by js87zz** [comment]
@Aryqs-Ipsum 
It's weird.. Can you give me the your `hook` function code?

**Comment by Aryqs-Ipsum** [comment]
Yes .. 

```js
methods: {
  async onImageUpload(blob, callback) {
    this.imageUploaderOpen = true
    const imageUrl = await this.$refs.uploader.getImageUrl(blob)
    callback(imageUrl, '')
  }
},
mounted() {
  this.$refs.toastuiEditor.editor.removeHook('addImageBlobHook')
  this.$refs.toastuiEditor.editor.addHook('addImageBlobHook', this.onImageUpload)
}
```

And I've add these lines to temporary fix the problem (in the hook after the callback) :
```js
this.editableContent = this.editableContent.replaceAll(BUCKET_URL + '/', BUCKET_URL + '%2F')
this.updateTextEditor()
```

**Comment by js87zz** [comment]
@Aryqs-Ipsum 
The bug has been fixed and released in [v3.0.1](https://github.com/nhn/tui.editor/releases/tag/editor%403.0.1).
